preview/Win7Msix/Win7MSIXInstallerLib/MsixRequest.cpp

```diff
@@ -168,7 +127,7 @@ HRESULT MsixRequest::ProcessAddRequest()
         HandlerInfo currentHandler = AddHandlers[currentHandlerName];
         AutoPtr<IPackageHandler> handler;
         RETURN_IF_FAILED(currentHandler.create(this, &handler));
-        RETURN_IF_FAILED(handler->ExecuteForAddRequest());
+        RETURN_IF_FAILED(handler->ExecuteForAddRequest(packageInfo, installationPath));

```

Comment:
>installationPath [](start = 68, length = 16)

if everything gets passed this, might as well make it part of the packageInfo?

Reply:
I would prefer to keep `PackageInfo` limited to its purpose (info about a package), but we can create another class `RequestInfo` containing a packageInfo and the path (+ other info in the future)